core_backend/app/contents/routers.py

```diff
@@ -35,7 +36,15 @@ async def create_content(
     upserts it to PG database
```

Comment:
Just noticed that it says "upserts" but it just "inserts" right?

Reply:
Yeah, will fix.